2-Authorization-I/1-call-graph/README.md

```diff
@@ -299,56 +295,53 @@ This sample app declares that it's CAE-capable by adding the `clientCapabilities
 
 #### Processing the CAE challenge from Microsoft Graph
```

Comment:
CAE related sections here should go after the discussion for how to acquire an access token. Basically,

Acquire a Token
Working with multiple resources
Dynamic scopes and incremental consent

and then the rest